Resources/Prototypes/Catalog/Fills/Boxes/syndicate.yml

```diff
@@ -102,4 +102,4 @@
         amount: 2
       - id: WeaponBaguette
       - id: SyndicateMicrowaveMachineCircuitboard
-      - id: PaperWrittenCombatBakeryKit
\ No newline at end of file
+      - id: PaperWrittenCombatBakeryKit
```

Comment:
untouch

Reply:
I didn't touch that line what-

Reply:
EOF newline

Reply:
So I need another one or to remove it, cause EOF has a newline

Reply:
You should remove it

Reply:
Done n dusted